css-pseudo-classes/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>css-psuedo-classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header><h1>Hover Button</h1></header>
+
+    <div>
+      <button id="hover-button">I'm a button!</button>
+    </div>
+
+    <h2>Striped Table</h2>
+    <div>
+      <table>
+        <thead class="table-header">
+          <tr>
+            <th>Student Name</th>
+            <th>Course</th>
+            <th>Grade</th>
+          </tr>
+        </thead>
+        <tbody class="odd-row">
+          <td>Ada Lovelace</td>
+          <td>Web Development</td>
+          <td>100</td>
+        </tbody>
+        <tbody class="even-row">
+          <td>Bob Martin</td>
+          <td>Web Development</td>
+          <td>100</td>
+        </tbody>
+        <tbody class="odd-row">
+          <td>Ryan Dahl</td>
+          <td>JavaScript</td>
+          <td>110</td>
+        </tbody>
+        <tbody class="even-row">
+          <td>Kent Beck</td>
+          <td>Cascading Style Sheets</td>
+          <td>90</td>
+        </tbody>
+        <tbody class="odd-row">
+          <td>Kyle Simpson</td>
+          <td>JavaScript</td>
+          <td>99</td>
+        </tbody>
```

Comment:
You should only have 1 `tbody` element. All direct children inside of the `tbody` should be a `tr`